src/controller/articlesController.js

```diff
@@ -324,13 +324,94 @@ class ArticleController {
         res.status(201).json({
           success: true,
           message: 'Article successfully bookmarked',
+          bookmark: createBookmark.dataValues,
         });
       }
     } catch (error) {
       helperMethods.serverError(res);
     }
   }
 
+  /**
+  * GET /api/v1/articles/bookmarks
+  * @description List Bookmarks
+  *
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  *
+  * @return {object} database response
+  * @memberof ArticleController
+ */
+  static async getBookmark(req, res) {
+    const userId = req.decoded.id;
+    try {
+      const bookmarks = await Bookmark.findAll({
+        where: { userId, isActive: true },
+        attributes: ['id'],
+        include: [{
+          model: Article,
+          as: 'bookmark',
+          attributes: ['title', 'body', 'readTime', 'image', 'createdAt'],
+          where: { isDeleted: false },
+          include: [{
```

Comment:
Why are you including the `User` model?

Reply:
So that I can get the Author's name of the Article.

Reply:
Okay